systemdspawner/systemdspawner.py

```diff
@@ -304,6 +319,11 @@ async def start(self):
             properties["CPUAccounting"] = "yes"
             properties["CPUQuota"] = f"{int(self.cpu_limit * 100)}%"
 
+        if self.cpu_weight is not None:
+            # FIXME: Detect & use proper properties for v1 vs v2 cgroups
```

Comment:
I've seen similar comments, but I don't know much about it, so I feel quite inactionable about these FIXME comments overall.

Are these systemd settings only valid in certain contexts with regards to available cgroup versions?